### ISSUE TITLE
Request: Pass parameter to pagination to disable it

### ISSUE THREAD

**Issue by RynoSvapos** [felixmosh/knex-paginate#25]
It would be very useful if you want to get all the records without pagination by passing a parameter. (and keep the same data structure)

**Comment by felixmosh** [comment]
Disable it how?
What data structure?

You can pass 99999999 as `perPage` and you will get the entire records... but sounds like you are trying to do something wrong.

**Comment by felixmosh** [comment]
Yeah, I get the idea, but it is weird to add a flag to disable the main functionality only for that JSON shape.

You achieve it like this:
```ts
const baseQuery = knex('tableName'); //without the `await` / call to `then`
const retObj = someFlagToDisablePagination
  ? { data: await baseQuery }
  : await baseQuery.paginate(...yourPaginationParams);
```

**Comment by RynoSvapos** [comment]
Thanks anyways (I thinked of this feature because it would require less function calls), but i'll refactor the code to look like this
